frontend/src/pages/Home/components/InitialSection/index.tsx

```diff
@@ -1,9 +1,42 @@
+import { useState } from "react";
+
+import { BtnContainer } from "@/components/Input/Radio";
+import { BtnRadio } from "@/components/Input/Radio/Btn";
+import { Select } from "@/components/Input/Select";
+import { Option } from "@/components/Input/Select/Option";
+
 import { InitialSectionContainer } from "./styles";
 
 export function InitialSection() {
+  const [selectedOption, setSelectedOption] = useState<string>("");
+
+  const handleOptionChange = (optionValue: string) => {
+    setSelectedOption(optionValue);
+  };
```

Comment:
Acredito que vamos ter que reestruturar isso aqui depois, se for necessário mesmo, faremos depois quando sentirmos necessidade no uso dos componentes.